Cubical/Algebra/Group/IsomorphismTheorems.agda

```diff
@@ -35,13 +36,20 @@ module _ {G H : Group ℓ} (ϕ : GroupHom G H) where
   open Iso
   open GroupTheory
 
-  private
-    kerϕ : NormalSubgroup G
-    kerϕ = kerSubgroup ϕ , isNormalKer ϕ
 
-    imϕ : Group ℓ
-    imϕ = imGroup ϕ
+  kerϕ : NormalSubgroup G
+  kerϕ = kerSubgroup ϕ , isNormalKer ϕ
+
+  imϕ : Group ℓ
+  imϕ = imGroup ϕ
```

Comment:
Should these really be public? Seems like something that is specific for this proof, or?

Reply:
If they are used elsewhere maybe have better names... `imGroup` is fine, but the kernel as a normal subgroup could have a nicer name